### PR TITLE
Added basic UI

### PR DESCRIPTION
**Pull request by NayanKaran**
Hi,

I have implemented the following project requirements. Kindly verify.

### Project requirements
- Watch the video with the UI example again (you have seen it while reading the Sneak Peek).
    - [UI example](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
- Your console app should behave in the same way as it is in the example.
- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    person_choice = gets.chomp.to_i
+    while person_choice.negative? || person_choice >= @people.length
+      print "Please enter a number within 0 - #{@people.length - 1} range: "
+      person_choice = gets.chomp.to_i
+    end
+    person = @people[person_choice]
+    print 'Enter date of booking: (yyyy/mm/dd) : '
+    date = gets.chomp.strip
+    person.add_rental(date, book)
+  end
```
Comment (prolajumokeoni) [inline app.rb]: -  Hi @NayanKaran I currently cannot create a rental, when I type in a number this is what I see ⬇️ 
Screenshot_414

- [x] Kindly make sure a user can create a rental, when I add a number the prompt is repeated it is not successful please fix this 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @NayanKaran, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (NayanKaran) [reply app.rb]: @prolajumokeoni Thank you for your suggestion. I have added the required corner case, https://github.com/NayanKaran/school-library/pull/5/commits/38c8b50c2738d1c5a29a87254fc84fd5a5b3a053

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @NayanKaran**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NayanKaran** [comment]
@BenMukebo Thank you for the code review. But I could not submit it now. Can you please help me? I am getting error message "Our systems run operations on Sundays, but since it is currently after program hours you cannot make progress towards your activities at this time. You will have full access to this page within the normal working hours of your scheduled program. Please check back at the start of your day."

**Pull request merged**